### PR TITLE
Update repository links to TypeStrong

### PR DESCRIPTION
**Pull request by blakeembrey**
A small start :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/jbrantly/ts-loader.svg?branch=master)](https://travis-ci.org/jbrantly/ts-loader)
+[![Build Status](https://travis-ci.org/jbrantly/ts-loader.svg?branch=master)](https://travis-ci.org/TypeStrong/ts-loader)
```
Comment (jbrantly) [inline README.md]: Need to change the path to the SVG as well

**Comment by jbrantly** [comment]
How do we want to do this? Merge your own after you get a +1? Let the other person merge?

File: README.md (hunk 1 above)
Comment (blakeembrey) [inline README.md]: Fix, thanks :+1:

**Comment by blakeembrey** [comment]
How about between collaborators, they merge their own PR after a LGTM and for other contributors we can merge after a review?

**Comment by jbrantly** [comment]
Saw your message literally 10 seconds after I merged :grin: Sounds good to me, will go with that in the future.

**Comment by blakeembrey** [comment]
Haha, it's all good - I don't think it's too important but if it's a new feature (not a patch) we could make sure there's at least two eyes on it to comment. E.g. that'll still be the same if you're the one that opens it since you're the maintainer :smile: I think if we both sign off it's not too important who merges, as long as it's passing.

**Pull request merged**